app/src/main/java/com/orange/ods/app/ui/components/sliders/ComponentSliders.kt

```diff
@@ -71,35 +78,55 @@ fun ComponentSliders() {
                 val rightIconContentDescription = if (hasSideIcons) stringResource(id = R.string.component_slider_high_volume) else null
 
                 var sliderPosition by remember { mutableStateOf(0f) }
+                val valueRange = 0f..100f
 
                 Title(textRes = getTitleRes(isStepped, hasSideIcons, shouldDisplayValue))
                 TechnicalText(text = technicalText)
 
                 Spacer(modifier = Modifier.padding(top = dimensionResource(R.dimen.spacing_m)))
 
+                val componentName: String
                 if (shouldDisplayValue) {
+                    componentName = OdsComponent.OdsSliderLockups.name
                     OdsSliderLockups(
                         value = sliderPosition,
                         steps = steps,
-                        valueRange = 0f..100f,
+                        valueRange = valueRange,
                         onValueChange = { sliderPosition = it },
                         leftIcon = leftIcon,
                         leftIconContentDescription = leftIconContentDescription,
                         rightIcon = rightIcon,
                         rightIconContentDescription = rightIconContentDescription
                     )
                 } else {
+                    componentName = OdsComponent.OdsSlider.name
                     OdsSlider(
                         value = sliderPosition,
                         steps = steps,
-                        valueRange = 0f..100f,
+                        valueRange = valueRange,
                         onValueChange = { sliderPosition = it },
                         leftIcon = leftIcon,
                         leftIconContentDescription = leftIconContentDescription,
                         rightIcon = rightIcon,
                         rightIconContentDescription = rightIconContentDescription
                     )
                 }
+
+                CodeImplementationColumn {
+                    ComposableCode(name = componentName, exhaustiveParameters = false, parameters = mutableListOf(
+                        StringRepresentationParameter("value", sliderPosition),
```

Comment:
This displays `value = 0.0` instead of `value = 0.0f`. We could add a `FloatParameter` class to handle this case.